### ISSUE TITLE
Initial tasks for writing the paper

### ISSUE THREAD

**Issue by LinguList** [lingpy/calc-workflow#6]
- [ ] assemble orthography profile from the inventories of Chen (provided by Doug Cooper)
- [ ] select as many languages overlapping in Chen and Ratliff for the demonstration and indicate which languages in Chen are Hmong and which are Mien
- [ ] translate the concept list into English
- [ ] refine the concepticon mapping (check for identical chinese terms in Huang 1992, where English terms can be found)
- [ ] select the concepts we want to use for the study (high overlap with concepts in Ratliff and in work of Nathan Hill on Burmish)
- [ ] restructure the paper following discussion from April 16 (suggestions by @lingulist)

**Comment by LinguList** [comment]
@MacyL and @schweikhard, you can see from this, that there are still quite a lot of things to be done here, specifically also concept mapping and orthoprofiles. Without these, we can't move forward with the paper, as it may make it difficult to have clear results. 

Note that orthography profile should cover all languages. The selection process will only be done later, and our argument for selecting less than all 25 languages is that we want to compare with Ratliff.

**Comment by MacyL** [comment]
I did the concepticon mapping for Chen's data when I convert it into CLDF format. I can translate it quickly now. But the concept list, you can already find it here : 
https://github.com/lexibank/chenhmongmien/blob/master/etc/concepts.tsv

**Comment by LinguList** [comment]
Yes, I know that. The idea is: 

1. make an automatic comparison of Chen2012 and Huang1992 for Chinese glosses. If they are identical, mark this, and assign Huang1992 English gloss.
2. fill in missing glosses, but use an official dictionary, to avoid misunderstandings (some styleguide, like "verb", "noun", etc.)

So this translation is something that should be planned, but starting point using Huang1992 (or also Sun1991) seems the most reasonable one, right?

**Comment by MacyL** [comment]
Reply to previous comment, I uploaded a file which I translated Chen's glosses to English by referencing Huang's data. The work is not done at this moment. I will work from home during the holiday, so I upload this file.

> Yes, I know that. The idea is:
> 
>     1. make an automatic comparison of Chen2012 and Huang1992 for Chinese glosses. If they are identical, mark this, and assign Huang1992 English gloss.
> 
>     2. fill in missing glosses, but use an official dictionary, to avoid misunderstandings (some styleguide, like "verb", "noun", etc.)
> 
> 
> So this translation is something that should be planned, but starting point using Huang1992 (or also Sun1991) seems the most reasonable one, right?

**Comment by LinguList** [comment]
Yes, this is indeed reasonable, and also helpful for the mapping and the tests!
